Sming/Makefile-rboot.mk

```diff
@@ -173,31 +179,7 @@ endif
 
 EXTRA_INCDIR += $(SMING_HOME)/include $(SMING_HOME)/ $(LWIP_INCDIR) $(SMING_HOME)/system/include $(SMING_HOME)/Wiring $(SMING_HOME)/Libraries $(SMING_HOME)/SmingCore $(SMING_HOME)/Services/SpifFS $(SDK_BASE)/../include $(THIRD_PARTY_DIR)/rboot $(THIRD_PARTY_DIR)/rboot/appcode $(THIRD_PARTY_DIR)/spiffs/src
 
-USER_LIBDIR  = $(SMING_HOME)/compiler/lib/
```

Comment:
The reason why there are all these changes is because I was having trouble bringing in the changes related to the debug messages with the rBoot makefile being so different from the default one. So I basically had to diff them and go change by change applying (or not) each one.

I wouldn't mind trying again but this requires testing on the hardware to see if it's fixed, and I no longer have the ESP8266 I was using with me, so this may have to wait some days.